src/cmd/services/m3query/config/config.go

```diff
@@ -497,6 +491,8 @@ type MiddlewareConfiguration struct {
 	Logging LoggingMiddlewareConfiguration `yaml:"logging"`
 	// Metrics configures the metrics middleware.
 	Metrics MetricsMiddlewareConfiguration `yaml:"metrics"`
+	// PrometheusRangeRewrite configures the range rewriting middleware.
+	PrometheusRangeRewrite PrometheusRangeRewriteMiddlewareConfiguration `yaml:"prometheusRangeRewrite"`
```

Comment:
nit: Should we create a `PrometheusMiddlewareConfiguration` struct instead of a specific range rewrite config? I feel like there may be future Prometheus fields here that might all need `prometheusFoo..`/`prometheusBar...` to share the same prefix at this level of config which is not always ideal.

Reply:
Good idea. I'll convert this